package.json

```diff
@@ -147,7 +147,8 @@
     "postcss": "8.4.31",
     "**/postcss": "8.4.31",
     "icss-utils": "5.0.0",
-    "**/icss-utils": "5.0.0"
+    "**/icss-utils": "5.0.0",
+    "@adobe/css-tools": "4.3.2"
```

Comment:
there wasn't a corresponding package-lock change too?

Reply:
added package lock.  npm install isn't updating the resolutions on my machine. i have to run npx force-resolutions